src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -1,6 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Authors>Confluent Inc.;Andreas Heider</Authors>
+    <Description>Confluent's .NET Client for Apache Kafka</Description>
+    <Copyright>Copyright 2016-2017 Confluent Inc., Andreas Heider</Copyright>
+    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
```

Comment:
license: librdkafka is not included in the package so we just need to point to the license in our repo (which includes the rdkafka-dotnet one). I've pointed to master, which is technically incorrect, but in practice this won't change and I don't want to update this link for every release.

summary + description: pity this tag doesn't exist. I like the current description for this. Leaving description short and sweat for now, which I think is fine and means I don't have to think hard about what I think should and shouldn't be in here and how to say it.

release notes: I just put a link to the github releases page, which is where I'll update release notes.